authentik/root/settings.py

```diff
@@ -302,18 +302,18 @@
         "TEST": {
             "NAME": CONFIG.get("postgresql.test.name"),
         },
+        # https://docs.djangoproject.com/en/4.0/ref/databases/#transaction-pooling-server-side-cursors
+        "DISABLE_SERVER_SIDE_CURSORS": CONFIG.get_bool(
+            "postgresql.disable_server_side_cursors", False
+        ),
+        # https://docs.djangoproject.com/en/4.0/ref/databases/#persistent-connections
+        # Not a PostgreSQL-specific setting, but a database-specific setting.
+        # Only PostgreSQL is supported as a database, so we place this setting it in the
+        # 'postgresql' path.
+        "CONN_MAX_AGE": CONFIG.get_optional_int("postgresql.conn_max_age", 0),
     }
 }
 
-if CONFIG.get_bool("postgresql.use_pgpool", False):
-    DATABASES["default"]["DISABLE_SERVER_SIDE_CURSORS"] = True
-
-if CONFIG.get_bool("postgresql.use_pgbouncer", False):
-    # https://docs.djangoproject.com/en/4.0/ref/databases/#transaction-pooling-server-side-cursors
-    DATABASES["default"]["DISABLE_SERVER_SIDE_CURSORS"] = True
-    # https://docs.djangoproject.com/en/4.0/ref/databases/#persistent-connections
-    DATABASES["default"]["CONN_MAX_AGE"] = None  # persistent
-
```

Comment:
We'll have an issue in the replica settings that get the data with `get` instead of `get_optional_int`. We probably need special cases in there.